### ISSUE TITLE
[Feature] projection switcher

### ISSUE THREAD

**Issue by nhoizey** [bravecow/mapbox-gl-controls#45]
Would it be possible to add a projection switcher to this project?

**Comment by korywka** [comment]
Yes but i don't know what UI should look like. Projection names are long words and it is easy to change it with `setProjection`: https://docs.mapbox.com/mapbox-gl-js/api/map/#map#setprojection

If you have some nice UI ideas, it would be helpful. Thanks.

**Comment by nhoizey** [comment]
It could be a simple button like the ruler and others, but open a list like for languages?
